stories/Pager.stories.mdx

```diff
@@ -21,37 +22,45 @@ export const DefaultPagerTemplate = (args) => html`
           <a aria-label="Go to page 4" href="javascript:void(0)">4</a>
       </li>
       <li>
-          <a>Next</a>
+          <a aria-label="Next" href="javascript:void(0)">Next<i class="jazz-icon jazz-icon-caret-right"></i></a>
```

Comment:
The aria-label should not be necessary here because the content is already "Next".  The label would only be necessary if we removed the text.